openedx/core/djangoapps/ccxcon/api.py

```diff
@@ -0,0 +1,146 @@
+"""
+Module containing API functions for the CCXCon
+"""
+
+import logging
+import urlparse
+from oauthlib.oauth2 import BackendApplicationClient
```

Comment:
Does edX strictly enforce the three import groups? It seems like some of these belong in the second group if so